example/record-mpi.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <mpi.h>
+
+#include <bigfile-mpi.h>
+
+static int BH_RECORD_POSITION = 0;
+static int BH_RECORD_VELOCITY = 0;
+static int BH_RECORD_BHMASS = 0;
+static BigRecordType BH_RECORD[1] = {{0}};
+
+void
```

Comment:
@sbird: What's your feeling about this style of API co-existing with the IO routines in MP-Gadget?

Reply:
From what I see, I really like this!